mrbgems/mruby-io/mrblib/io.rb

```diff
@@ -184,6 +184,11 @@ def ungetc(substr)
     nil
   end
 
+  def ungetbyte(substr)
+    substr = substr.chr if substr.is_a? Integer
```

Comment:
`Integer#chr` is defined in `mruby-string-ext`.
So it needs to add a dependency.